src/state/selectors/index.js

```diff
@@ -1,72 +1,82 @@
+import { createSelector } from 'reselect';
 import filter from 'lodash/filter';
 import flatten from 'lodash/flatten';
-import { LanguageMap } from 'manifesto.js';
+import manifesto, { LanguageMap } from 'manifesto.js';
 import Annotation from '../../lib/Annotation';
 import ManifestoCanvas from '../../lib/ManifestoCanvas';
 import CanvasGroupings from '../../lib/CanvasGroupings';
 
-/**
-* Return the manifest that belongs to a certain window.
-* @param {object} state
-* @param {String} windowId
-* @return {object}
-*/
-export function getWindowManifest(state, windowId) {
-  return state.windows[windowId]
-    && state.windows[windowId].manifestId
-    && state.manifests[state.windows[windowId].manifestId];
+/** Get the relevant manifest information */
+export function getManifest(state, { manifestId, windowId }) {
```

Comment:
This seems to return a manifest by either a `manifestId` or a `windowId`? Does it matter if two windows are using the same manifest?

Reply:
I don't think so. I guess we might not reuse the same manifesto instance for each window, but I'm not sure we care?